contracts/migrations/4_user_replica_set_migration.js

```diff
@@ -76,5 +72,20 @@ module.exports = (deployer, network, accounts) => {
     // Confirm registered address matches proxy
     let retrievedAddressFromRegistry = await registry.getContract(userReplicaSetManagerKey)
     console.log(`Registered ${retrievedAddressFromRegistry} with key ${userReplicaSetManagerKeyString}/${userReplicaSetManagerKey}`)
+
+    // Confirm seed is not yet complete
+    let userReplicaSetManagerInst = await UserReplicaSetManager.at(deployedProxyTx.address)
+    let seedComplete = await userReplicaSetManagerInst.getSeedComplete({ from: userReplicaSetBootstrapAddress })
+    console.log(`Seed complete: ${seedComplete}`)
+    // Issue seed operation
+    // TODO: REMEMBER THAT THIS CANNOT BE CALLED FROM SAME WALLET as deployer
```

Comment:
could you explain more what this means?

Reply:
You cannot call logic functions from the proxy admin account - they end up falling back to the proxy functions like `upgradeTo` etc

Reply:
This has to be called from a separate account basically

Reply:
i think i see where i got confused. in the contract the variable userReplicaSetBootstrapAddress is actually the on permissioned to create/update replica sets. the deployer has no special permissions

Reply:
also - since we know we plan to call it from the bootstrapaddress why not just restrict caller to this in the function and then not worry about this. doesn't really matter tho

Reply:
^I agree with this latest statement its much better to restrict to userReplicaBootstrapAddress